app/models/publisher.rb

```diff
@@ -92,6 +92,10 @@ class Publisher < ApplicationRecord
     .where("uphold_updated_at < ?", UPHOLD_ACCESS_PARAMS_TIMEOUT.ago)
   }
 
+  scope :has_verified_channel, -> {
```

Comment:
would you rename to `verified_channels`? `has_verified_channel` implies a boolean return type as opposed to a collection

Reply:
how about `with_verified_channels`?